### PR TITLE
New version: VNGCorp.Zalo version 23.7.1

### PR DESCRIPTION
**Pull request by zeldrisho**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/114328&drop=dogfoodAlpha

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

File: manifests/v/VNGCorp/Zalo/23.7.1/VNGCorp.Zalo.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created with Komac v1.9.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: VNGCorp.Zalo
+PackageVersion: 23.7.1
+InstallerType: nullsoft
+Installers:
+- Architecture: x86
+  InstallerUrl: https://res-download-pc-te-vnso-tt-10.zadn.vn/win/ZaloSetup-23.7.1.exe
```
Comment (stephengillie) [inline manifests/v/VNGCorp/Zalo/23.7.1/VNGCorp.Zalo.installer.yaml:9]:
```suggestion
  InstallerUrl: https://res-zaloapp-aka.zdn.vn/win/ZaloSetup-23.7.1.exe
```

Comment (stephengillie) [reply manifests/v/VNGCorp/Zalo/23.7.1/VNGCorp.Zalo.installer.yaml:9]: Hi @giatrung2012,

This is the InstallerUrl I find from opening the PackageUrl. Is this the same package?

Comment (giatrung2012) [reply manifests/v/VNGCorp/Zalo/23.7.1/VNGCorp.Zalo.installer.yaml:9]: Yes, this is the same package.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**